app/controllers/webhooks_controller.rb

```diff
@@ -8,6 +8,8 @@ class WebhooksController < ApplicationController
 
   def index
     @webhooks = current_project.webhooks
+    @outbound_webhooks = current_project.outbound_webhooks
+    @new_outbound_webhook = OutboundWebhook.new
```

Comment:
why does not normal webhook not need an instance variable ?
... can we move the instance variables into the view to have less duplication ?